docs/content/sre/tronador/tronador_config.md

```diff
@@ -136,25 +136,75 @@ spec:
 - **`authSecret`**: Name of the Secret required in case of a private repository. If this field is not filled, it will be treated as a public repository.
 - **`imagePullSecret`**: Name of the Secret required to access the image repository where the application's images are hosted, e.g. Nexus or Docker.
 
+Note: The Secrets mentioned under `mappings.authSecret` and `mappings.imagePullSecret` will only be created in the relevant Environments' namespaces, and not in any other namespaces. The relevant Environments are those that are based on the repositories mentioned in `mappings.repos`.
```

Comment:
pls shift it to the last subheading `## Points to be considered while applying TronadorConfig`

Reply:
i would want it to be immediately next to the explanation, so that poeple who ignore the rest of the page see it too

Reply:
it already does explain it here `A list of repositories on whose DTEs the mapping will apply` and its not an immediate action item for the user, since its `tronador` config.

Reply:
true, the user does not need to take any extra caution. moving it